app/src/main/java/com/mensinator/app/business/MensinatorExportImport.kt

```diff
@@ -89,7 +89,7 @@ class MensinatorExportImport(
         return jsonArray
     }
 
-    override fun importDatabase(filePath: String) {
+    override fun importDatabase(filePath: String) : Boolean {
```

Comment:
This method does not use a catch block inside `try`, the behavior is thus different. This method currently throws!

I would suggest to also catch exceptions here, so that all methods behave the same way.

Reply:
Then at SettingsViewModel#handleImport, you can remove the try catch block.